test-runner-poc/config.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+const f = require('util').format;
+const url = require('url');
+const qs = require('querystring');
+const core = require('../lib/core');
+
+class ConfigurationBase {
+  constructor(options) {
+    this.options = options || {};
+    this.host = options.host || 'localhost';
+    this.port = options.port || 27017;
+    this.db = options.db || 'integration_tests';
+    this.manager = options.manager;
```

Comment:
It seems like we shouldn't be using `manager` at all anymore since we're removing the dependencies on `topologyManager`.